gap_quantization/dump_utils.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os
+import os.path as osp
+import re
+import shutil
+
+import torch
+import torch.nn as nn
+
+#from urllib.parse import urlparse
+
+
+def dump_quant_params(save_dir, convbn):
+    dict_norm = {}
+    for file in os.listdir(save_dir):
+        if file != 'activations_dump' and not re.match('.*cat.json', file) and not re.match('.*txt', file):
+            dict_norm[file] = read_norm(os.path.join(save_dir, file))
+    list_norm = make_list_from_dict(dict_norm, convbn)
+    txt_list = open('param_layer_quant.h', 'w')
+    for i in range(0, 26):
+        txt_list.write("#define Q_IN_" + '{:<8}'.format(str(i)) + str(list_norm[i]['q_in']) + "\n")
+        txt_list.write("#define Q_OUT_" + '{:<7}'.format(str(i)) + str(list_norm[i]['q_out']) + "\n")
+        txt_list.write("#define Q_WEIGHTS_" + '{:<3}'.format(str(i)) + str(list_norm[i]['q_weights']) + "\n")
+        txt_list.write("#define Q_BIAS_" + '{:<6}'.format(str(i)) + str(list_norm[i]['q_bias']) + "\n")
+        txt_list.write("\n")
+
+
+def read_norm(file):
+    print(file)
+    with open(file, "rt") as js_file:
```

Comment:
`read` and `text` are default modes for `open()`
also `js_file` is not a good name, as it's not related to javascript